3rdparty/CMakeLists.txt

```
@@ -493,6 +497,27 @@ list(APPEND EP_TARGETS ${RE2_TARGET})
 list(APPEND EP_LIBS ${RE2_LIBS})
 list(APPEND EP_INCLUDES ${RE2_INCLUDES})
 
+# HWLOC
+set(HWLOC_TARGET hwloc)
+set(HWLOC_INCLUDES ${3RDPARTY_LIBS_DIR}/Install/hwloc/include)
+set(HWLOC_LIBS ${3RDPARTY_LIBS_DIR}/Install/hwloc/lib/libhwloc.a)
+ADD_BY_PRODUCTS_VARIABLE ("HWLOC" ${HWLOC_LIBS})
+externalproject_add(${HWLOC_TARGET}
+  URL                   ${WITH_HWLOC_URL}
+  URL_HASH              ${WITH_HWLOC_URL_HASH}
+  TIMEOUT               600
+  LOG_BUILD             TRUE
+  DOWNLOAD_NO_PROGRESS  1
+  CONFIGURE_COMMAND     ./configure --prefix=${3RDPARTY_LIBS_DIR}/Install/hwloc --enable-static --disable-shared
```

libxml 관련한 의존성 에러가 발생해서 다음과 같이 수정하니 해결되었습니다


```suggestion
  CONFIGURE_COMMAND     ./configure --prefix=${3RDPARTY_LIBS_DIR}/Install/hwloc --disable-libxml2 --enable-plugins=xml_libxml --enable-static --disable-shared
```

---

다음 처리에서 함께 반영하겠습니다